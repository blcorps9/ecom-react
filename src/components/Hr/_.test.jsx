import React from "react";
import renderer from "react-test-renderer";

import Hr from "./index";

describe("<Hr />", () => {
  it("renders successfully", () => {
    const component = renderer.create(<Hr />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
