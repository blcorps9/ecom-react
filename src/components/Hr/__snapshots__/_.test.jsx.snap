// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hr /> renders successfully 1`] = `
<span
  className="hr"
/>
`;
